resources/views/components/menu/account.blade.php

```diff
@@ -46,6 +46,9 @@
         </x-slot>
         <x-dropdown-header>{{ $user->username }}</x-dropdown-header>
         <x-dropdown-item :link="route('user.show', $user)">{{ __res('profile', 1) }}</x-dropdown-item>
+        @if($user->Permissions >= Permissions::Registered)
+            <x-dropdown-item :link="url('gameList.php?t=play')">Want to Play Games</x-dropdown-item>
```

Comment:
Do you think this should read "Want to Play List" to match the messaging of the API response labels?

![Screenshot 2023-09-02 at 10 03 28 AM](https://github.com/RetroAchievements/RAWeb/assets/3984985/f240335c-d24b-401b-b23b-15d215237309)

![Screenshot 2023-09-02 at 10 03 23 AM](https://github.com/RetroAchievements/RAWeb/assets/3984985/5c76d8d0-3d09-476e-9523-164659db31b0)

Reply:
I've updated the messages to say "Want to Play Games", as that's a better title for the page, and therefore the menu.